### ISSUE TITLE
add __version__ function into mrob python bindings

### ISSUE THREAD

**Issue by nosmokingsurfer** [MobileRoboticsSkoltech/mrob#67]
It would be great to have a function which returns the version of mrob python bindings:

For example:
`import mrob
mrob.__version__
`
should return something like:
`0.0.3`

All popular python libraries do so. For example:

`import numpy`
`numpy.__version__`
'1.15.4'
>>>

**Comment by nosmokingsurfer** [comment]
@anastasiia-kornilova 

I've checked the usual ways how to add this attribute to the package:
https://stackoverflow.com/questions/41146784/set-python-wheel-version-tag-dynamically
https://stackoverflow.com/questions/17791481/creating-a-version-attribute-for-python-packages-without-getting-into-troubl

Besides the ```__version__``` there are a few other attributes:
https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

It makes sense to add them all.

It should be done in ```setup.py```

**Comment by anastasiia-kornilova** [comment]
@nosmokingsurfer , I'm not sure that those stackoverflow straightforward solutions are the most beatiful ones. But they are right about `setup.py`.

Actually, as you may notice, we already use `setuptools-git-versioning` to automatically version pip-package (https://pypi.org/project/setuptools-git-versioning/).

You can find this in toml project file -- https://github.com/MobileRoboticsSkoltech/mrob/blob/f08e88c98bb7703f34504616fabf7fdb1755238a/pyproject.toml#L24
And then it is used in setup.py --  https://github.com/MobileRoboticsSkoltech/mrob/blob/f08e88c98bb7703f34504616fabf7fdb1755238a/setup.py#L55

In manual of this tool (https://pypi.org/project/setuptools-git-versioning/) in section "Callback/variable as current version" you could find how to use this tool to assign value to `__version__` variable of the python package. I would try this solution first.

**Comment by anastasiia-kornilova** [comment]
@nosmokingsurfer , CI -- is your friend, it contains all steps in a reproducible way in comparison to any other guide/manual :)

Follow `build-wheel` step -- https://github.com/MobileRoboticsSkoltech/mrob/blob/f08e88c98bb7703f34504616fabf7fdb1755238a/.circleci/config.yml#L63 -- install proper docker container, install packages inside it, attach your sources instead of `checkout` and run  `scripts/build-wheels.sh`.

**Comment by anastasiia-kornilova** [comment]
@nosmokingsurfer , I've realized that the step you propose will change nothing, it just adds tag to the pip-package and doesn't change the attribute `__version__` . This tag we already have with `version_config=True`. What we need is to assign to `__version__` current pip-package distribution version in `__init__.py`:

```
from pkg_resources import get_distribution
__version__ = get_distribution('mrob').version

Btw, instead of building things locally I would propose to use artifacts build on CI -- there is a special section where you can download built pip-wheel and install locally. Just commit changes on some branch and visit CI page in one minute.

**Comment by nosmokingsurfer** [comment]
@anastasiia-kornilova 

Adding your lines in `__init__.py` worked straight. I used CI artefacts to test the wheels, as you recommended. Here is the current output:
![Screenshot from 2021-07-13 17-38-51](https://user-images.githubusercontent.com/22825575/125471973-bc560f02-4090-42f7-9f54-add29d7e2aae.png)

Is it ok or should it only report the digital part of the version? I don't think that the information about what git state was used to build the wheel is required, it already contained in the name of the `whl` file

**Comment by anastasiia-kornilova** [comment]
@nosmokingsurfer , I would say that it is the standard way in common practice (that is why it is the default behavior for setuptools package) -- we need to separate "release" version and some "dirty/experimental" version. The way you propose -- after the installation env doesn't remember what wheel was used for installation (as well as a user), and this could produce some ambiguities. So, I propose to remain it as it is now.

Also, you were proposing to add other attributes, I suppose they could be added in the same way as with the version -- wheel already encapsulates information about maintainer/licence/etc.

**Comment by nosmokingsurfer** [comment]
@anastasiia-kornilova 
You mean, if the wheel will correspond to some release tag, then the version value will automatically become a "pretty" release version like "0.0.7"? If so, then ok.

About other attributes, yes, sure. I will add them as soon as I understand how it worked exactly in the case of the version attribute =)  I've never done package wrapping before, just need to read the documentation more and play around with tools for packaging.

**Comment by anastasiia-kornilova** [comment]
About versioning -- yes, it should be so because `setuptools-git-versioning` state that they follow PEP-440. In their README:

`Use git repo data (latest tag, current commit hash, etc) for building a version number according PEP-440.`

I hope it works, you could check CI artifacts from commits that were tagged and output version of it using `pkg_resources` directly.

Sure, any questions about pip-package are welcome, many thanks to your contributions!

**Comment by anastasiia-kornilova** [comment]
Or even check "release" version from our releases -- https://github.com/MobileRoboticsSkoltech/mrob/releases

**Comment by nosmokingsurfer** [comment]
I've looked at the ```numpy``` and ```pandas``` packages as a reference. It makes sense to add the following dunder names to add to the package:
- [ ] __version__
- [ ] __git_version__

**Comment by nosmokingsurfer** [comment]
Metadata for mrob improved:

Before:
![Screenshot from 2021-11-08 01-30-30](https://user-images.githubusercontent.com/22825575/140664213-472e4816-19ec-49a5-bdb1-821449d7cbaf.png)

Now:
![Screenshot from 2021-11-08 01-21-53](https://user-images.githubusercontent.com/22825575/140664107-97969ef4-b4d8-4747-8e57-9c82e0fed4be.png)

- branch added into the version value + possible to set full SHA value if required (setuptools-git-versioning);
- summary added;
- author and author e-mail added;
- download URL added;
- supported platforms added = 'any';

MROB package now has attribute __version__ that can be called from the console line:
![Screenshot from 2021-11-08 01-49-07](https://user-images.githubusercontent.com/22825575/140664759-34ac8d68-4e79-44a3-90a2-c09a312123b5.png)

Git full SHA can be also enabled if required, but having branch name in __version__ is more informative during development.

**Comment by anastasiia-kornilova** [comment]
Closed in #88
